lib/mobility/plugins/callback_on_write.rb

```diff
@@ -0,0 +1,33 @@
+require 'mobility'
+require 'i18n_sonder/callback_on_write'
+
+module Mobility
+  module Plugins
+    module CallbackOnWrite
+      class << self
+        def apply(attributes, option)
+          return unless option
+          # include the write instance method so that we override the backend's write method
+          attributes.backend_class.include self
+        end
+      end
+
+      def write(locale, value, options = {})
+        return unless should_execute_callback?(value, attribute, locale)
+        
+        super
+        I18nSonder::CallbackOnWrite.trigger.call(model, locale)
+      end
+
+      private
+
+      # Only execute callback if:
+      # 1) the new value being written is different to the already existing value
+      def should_execute_callback?(value, attribute, locale)
```

Comment:
Don't we only want to execute the callback when the locale is not the default_locale? Or we should have that as an option, since that's kind of what we need to do for the case of pushing translations.

Reply:
missed the condition at the end of line 29, could you augment the function comment to include that condition as well?